src/components/sbb-tooltip/sbb-tooltip.tsx

```diff
@@ -399,23 +402,7 @@ export class SbbTooltip implements ComponentInterface {
         ? this._nextFocusedElement
         : this._triggerElement;
 
-      const closedByModality = sbbInputModalityDetector.mostRecentModality;
-
-      // Set focus origin to element which should receive focus
-      if (elementToFocus && closedByModality !== null) {
-        elementToFocus.addEventListener(
-          'focus',
-          () => {
-            (elementToFocus.dataset.focusOrigin as SbbInputModality) = closedByModality;
-            elementToFocus.addEventListener(
-              'blur',
-              () => delete elementToFocus.dataset.focusOrigin,
-              { once: true }
-            );
-          },
-          { once: true }
-        );
-      }
+      focusAndSetCloseModality(elementToFocus);
```

Comment:
I think we should move this after close()

Reply:
If I move it after close I noticed that it doesn't work anymore...